saleor/plugins/base_plugin.py

```diff
@@ -33,12 +34,14 @@ class ConfigurationTypeField:
     STRING = "String"
     BOOLEAN = "Boolean"
     SECRET = "Secret"
+    SECRET_TEXT = "SecretText"
     PASSWORD = "Password"
     CHOICES = [
         (STRING, "Field is a String"),
         (BOOLEAN, "Field is a Boolean"),
         (SECRET, "Field is a Secret"),
         (PASSWORD, "Field is a Password"),
+        (SECRET_TEXT, "Field is a Secret multiline"),
```

Comment:
Maybe `SECRET_MULTILINE` would be a better name? Just wondering, `SECRET` and `SECRET_TEXT` sounds the same to me.